components/tidb_qe/src/executor/aggregation.rs

```diff
@@ -7,15 +7,18 @@ use std::sync::Arc;
 use tipb::executor::Aggregation;
 use tipb::expression::{Expr, ExprType};
 
-use tikv_util::collections::{OrderMap, OrderMapEntry};
+use indexmap::map::Entry as OrderMapEntry;
+use indexmap::IndexMap as OrderMap;
 
 use super::aggregate::{self, AggrFunc};
 use super::{Executor, ExprColumnRefVisitor, Row};
-use crate::coprocessor::codec::datum::{self, Datum};
-use crate::coprocessor::dag::execute_stats::ExecuteStats;
-use crate::coprocessor::dag::expr::{EvalConfig, EvalContext, EvalWarnings, Expression};
-use crate::coprocessor::dag::storage::IntervalRange;
-use crate::coprocessor::*;
+use crate::codec::datum::{self, Datum};
+use crate::execute_stats::ExecuteStats;
+use crate::expr::{EvalConfig, EvalContext, EvalWarnings, Expression};
+use crate::storage::IntervalRange;
+use crate::Result;
+
+const SINGLE_GROUP: &[u8] = b"SingleGroup";
```

Comment:
Should we give a shorter key for this?

Reply:
We can keep it untouched for now. It shall be removed in DAG v2 to implement a TiDB compatible schema.